packages/wrangler/src/deploy/deploy.ts

```diff
@@ -885,7 +877,6 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 								headers: await getMetricsUsageHeaders(config.send_metrics),
 							},
 							new URLSearchParams({
-								include_subdomain_availability: "true",
```

Comment:
should we remove https://github.com/cloudflare/workers-sdk/blob/481445571735978b6af036254a82b3b9ca73f161/packages/wrangler/src/versions/secrets/index.ts#L219 as well?

Reply:
I'll not remove for now just to keep the scope a bit smaller but yeah, it isn't needed here either